e2e-tests/tests/custom-extensions/02-assert.yaml

```diff
@@ -35,15 +35,19 @@ spec:
           value: s3
         - name: STORAGE_ENDPOINT
         - name: STORAGE_REGION
-          value: eu-central-1
+          value: us-east-1
         - name: STORAGE_BUCKET
-          value: pg-extensions
+          value: operator-testing
```

Comment:
Maybe it's better to use separate bucket to store extensions.

Reply:
@nmarukovich it is minio deployment